R/mlflow/R/connection.R

```diff
@@ -151,10 +151,14 @@ new_mlflow_connection <- function(tracking_uri, handle, ...) {
 #'
 #' Disconnects from a local MLflow instance.
 #'
-#' @param mc The MLflow connection created using \code{mlflow_connect()}.
 #' @export
-mlflow_disconnect <- function(mc) {
-  if (mc$handle$is_alive()) invisible(mc$handle$kill())
+mlflow_disconnect <- function() {
+  mc <- mlflow_active_connection()
+  if (is.null(mc)) message("Not connected to an MLflow service.") else {
```

Comment:
I added a linter to match `mlflow`, I think it will complain that all `if` statements need to be written as:

```
if () {
}
else {
}
```

If this bother us enough we can change the `.lintr` file and turn this off for the ones that we don't agree with,